app/models/group_offer_category.rb

```diff
@@ -1,8 +1,17 @@
 class GroupOfferCategory < ApplicationRecord
   has_many :group_offers
+  has_and_belongs_to_many :volunteers
```

Comment:
I tried to do it, but somehow only threw me errors and I assume I would need to refactor then also all forms.. since we dont have any other attributes, validations etc I will leave it like this until it is requested to add new stuff to it.